src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -157,6 +157,9 @@ class AmendmentTableImpl final
     // we haven't participated in one yet.
     std::unique_ptr <AmendmentSet> lastVote_;
 
+    // True if an unsupported amendment is enabled
+    bool missingSupport_;
```

Comment:
Consider renaming this and the accessor method `unsupportedEnabled`.   To me, missing support sounds like a potential amendment has yet to receive enough votes by the network and is therefore missing support.